curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/6633b4d3271d5b3aea1c1195.md

```diff
@@ -0,0 +1,172 @@
+---
+id: 6633b4d3271d5b3aea1c1195
+title: Step 13
+challengeType: 20
+dashedName: step-13
+---
+
+# --description--
+
```

Comment:
This may be a bit too difficult.

Reply:
What would you suggest should be the smaller step?